src/components/libraryoptionseditor/libraryoptionseditor.js

```diff
@@ -416,6 +416,12 @@ export function setContentType(parent, contentType) {
         }
     }
 
+    if (contentType === 'music') {
+        parent.querySelector('.chkEnableLUFSScan').classList.remove('hide');
+    } else {
+        parent.querySelector('.chkEnableLUFSScan').classList.add('hide');
+    }
```

Comment:
I've been suggesting people use `classList.toggle` in new code because it is easier to read imo

```suggestion
    parent.querySelector('.chkEnableLUFSScan').classList.toggle('hide', contentType !== 'music');
```

Reply:
Okay that is a lot nicer to read :)